CONTRIBUTING.md

```diff
@@ -45,17 +45,59 @@ Ariadne implements [subscriptions-transport-ws](https://github.com/apollographql
 
 ### 5. Production server
 
+As of 12/15/2022, the production LANE-server may only be accessed on the LANL OCE network at http://nedm-macpro.lanl.gov/graphql/
+
 The production server has Python 3.6.9 and Node js 16.15.1
 
-### 6. Staging
+To update LANE-server on production, perform the following:
+
+```bash
+# Assuming you are in the $HOME/LANE/LANE-server directory
+chmod u+x deploy-prod/deploy.sh
+./deploy-prod/deploy.sh # Doesn't matter which directory from which you call deploy.sh
+```
```

Comment:
I don't think this comment is necessary. The example assumes user is in `$HOME/LANE/LANE-server` and that alone should suffice.

Reply:
Comment was removed